src/redis/zmalloc.h

```diff
@@ -82,32 +84,39 @@
 #define HAVE_DEFRAG
 #endif
 
-void *zmalloc(size_t size);
-void *zcalloc(size_t size);
-void *zrealloc(void *ptr, size_t size);
-void *ztrymalloc(size_t size);
-void *ztrycalloc(size_t size);
-void *ztryrealloc(void *ptr, size_t size);
-void zfree(void *ptr);
+void* zmalloc(size_t size);
```

Comment:
please do not reformat files under redis dir.

Reply:
ok, I should not use the IDE for these files ..